softwarereview_editor_management.es.Rmd

```diff
@@ -1,95 +1,98 @@
 
-* Invite editors to the rOpenSci GitHub organization as member, as a member of the [`editors` team](https://github.com/orgs/ropensci/teams/editors) and the [`data-pkg-editors`](https://github.com/orgs/ropensci/teams/data-pkg-editors) or [`stats-board`](https://github.com/orgs/ropensci/teams/stats-board) sub-team, as appropriate.  This will give them appropriate permissions and allow them to get team-specific notifications. 
+- Los editores necesitan acceso a la base de datos AirTable de revisión de software.
 
-* Editors need access to the AirTable database of software review.
+- Los editores necesitan acceso al canal privado de editores en el espacio de trabajo de Slack de rOpenSci (y al espacio de trabajo de Slack en general si no lo tenían previamente, en ese caso pregunta al gestor de la comunidad de rOpenSci).
 
-* Editors need access to the private editors channel in rOpenSci Slack workspace (and to the Slack workspace in general if they didn't previously, in that case ask rOpenSci community manager).
+- Publica un mensaje de bienvenida en el canal, haciendo ping a todos los editores.
 
-* Post a welcome message in the channel, pinging all editors.
+- En el espacio de trabajo de Slack tienen que ser añadidos al equipo de editores para que `@editors` también se les enviará un ping.
 
-* In the Slack workspace they need to be added to the editors team so that `@editors` will ping them too.
+- Añadimos los nombres de los editores a
+  
+  - [lista de autores de dev\_guide](https://github.com/ropensci/dev_guide/blob/main/index.Rmd)
+  - [capítulo dev\_guide que introduce la revisión del software](https://github.com/ropensci/dev_guide/blob/main/softwarereview_intro.Rmd) (en dos lugares de este archivo, como editores y un poco más abajo para eliminarlos de la lista de revisores)
+  - [revisión del software README](https://github.com/ropensci/software-review/blob/main/README.Rmd) (también en dos lugares de este archivo)
+    Tanto el dev\_guide como el software-review README se tejen automáticamente a través de la integración continua.
 
-* We add editors' names to 
-    * [dev_guide authors list](https://github.com/ropensci/dev_guide/blob/main/index.Rmd)
-    * [dev_guide chapter introducing software review](https://github.com/ropensci/dev_guide/blob/main/softwarereview_intro.Rmd) (at two locations in this file, as editors and a bit below to remove them from the reviewers list)
-    * [software-review README](https://github.com/ropensci/software-review/blob/main/README.Rmd) (in two places in this file as well)
-Both the dev_guide and software-review README are automatically knit via continuous integration.
+- Añade editores a [https://github.com/orgs/ropensci/teams/editors/members](https://github.com/orgs/ropensci/teams/editors/members)
 
-* Add editors to https://github.com/orgs/ropensci/teams/editors/members
```

Comment:
I think this is redundant since it's mentioned in line 61. I can delete it from the translation and open an issue / PR to change it in the original.

Reply:
@maelle what do you think?

Reply:
ok, sounds good, thank you!